source/05-EL/02.ptx

```diff
@@ -12,7 +12,840 @@
+                    p
+                  </sageplot>
+                </image> </p>  </li>
+
+                <li> <p> <image>
+                  <sageplot>
+                    f(x) = (4)^(x+1)
+                    p=point([(-3,1/8),(-2,1/4),(-1,1),(0,4),(1,16)],pointsize=50,color='blue')
+                    p+=plot(f, (x, -9, 8), ymin=-2, ymax=20, color='blue', thickness=3, gridlines=[[-8,-7,..,8],[-8,-7,..,20]])
+                    p
+                  </sageplot>
+                </image> </p>  </li>
+
+                <li> <p> <image>
+                  <sageplot>
+                    f(x) = -4^(x)
+                    p=point([(-2,-1/8),(-1,-1/4),(0,-1),(1,-4),(2,-16)],pointsize=50,color='blue')
+                    p+=plot(f, (x, -9, 8), ymin=-20, ymax=2, color='blue', thickness=3, gridlines=[[-8,-7,..,8],[-20,-19,..,2]])
+                    p
+                  </sageplot>
+                </image> </p>  </li>
+              
+              
+            </ol>
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <ul>
+              <li> <p><m> g(x)= -4^x </m> is D</p> </li>
+              <li> <p><m> h(x)= 4^{-x} </m> is B</p> </li>
+              <li> <p><m> j(x)= 4^{x+1} </m> is C</p> </li>
+              <li> <p><m> k(x)= 4^x +1 </m> is A</p> </li>
+            </ul>
+          </p>
+        </answer>
+      </task>
+    </activity>
+
+    <activity xml:id="activity-exp-e">
+  
+  <task>
+    <statement>
+      <p>
+        Graph  <m>f(x)=e^{x}</m>. First find <m>f(0)</m> and <m>f(1)</m>.  Then use what you know about the characteristics of exponential graphs to sketch the rest.(Recall that <m>e \approx 2.72</m> to help estimate where to put your points.) 
+      </p>
+    </statement>
+    <answer>
+      <p>
+        <image>
+          <sageplot>
+            f(x) = e^(x)
+            p=point([(0,1),(1,e)],pointsize=50,color='blue')
+            p+=plot(f, (x, -9, 8), ymin=-2, ymax=8, color='blue', thickness=3, gridlines=[[-8,-7,..,8],[-8,-7,..,8]])
+            p
+          </sageplot>
+        </image>
+      </p>
+    </answer>
+  </task>
+
+  <task>
+    <statement>
+      <p>
+        Sketch the graph of <m>g(x)=e^{x-2}</m> using transformations. State the transformation(s) used.
+      </p>
+    </statement>
+    <answer>
+      <p>
+        <image>
+          <sageplot>
+            f(x) = e^(x-2)
+            p=point([(2,1),(3,e)],pointsize=50,color='blue')
+            p+=plot(f, (x, -9, 8), ymin=-2, ymax=8, color='blue', thickness=3, gridlines=[[-8,-7,..,8],[-8,-7,..,8]])
+            p
+          </sageplot>
+        </image>
+      </p>
+    </answer>
+  </task>
+
+  <task>
+    <statement>
+      <p>
+        Sketch the graph of <m>h(x)=3e^x</m> using transformations. State the transformation(s) used.
+      </p>
+    </statement>
+    <answer>
+      <p>
+        <image>
+          <sageplot>
+            f(x) = 3*e^x
+            p=point([(0,3),(1,3e)],pointsize=50,color='blue')
```

Comment:
```suggestion
            p=point([(0,3),(1,3*e)],pointsize=50,color='blue')
```

Reply:
Or, more cutely,

```suggestion
            p=point([(0,f(x=0)),(1,f(x=1))],pointsize=50,color='blue')
```

Reply:
Omg. I played around with the f(x) part so much trying to figure out the right combo to make it work and I forgot I had points in there too that might be the problem. 🤦‍♀️

Thanks!

Reply:
yeah, it's a shame there isn't an API like `f.point(x=0)` to get the point object, rather than constructing it. Ah well.

If you like that fix, click "Commit suggestion" above and it'll automatically make a commit with that change for you.